internal/service/console.go

```diff
@@ -0,0 +1,40 @@
+package service
+
+import (
+	"context"
+
+	"github.com/flightctl/flightctl/internal/api/server"
+	"github.com/flightctl/flightctl/internal/flterrors"
+	"github.com/flightctl/flightctl/internal/store"
+	"github.com/google/uuid"
+)
+
+func (h *ServiceHandler) RequestConsole(ctx context.Context, request server.RequestConsoleRequestObject) (server.RequestConsoleResponseObject, error) {
+	orgId := store.NullOrgId
+
+	// make sure the device exists
+	_, err := h.store.Device().Get(ctx, orgId, request.Name)
+	if err != nil {
+		switch err {
+		case flterrors.ErrResourceNotFound:
+			return server.RequestConsole404JSONResponse{}, nil
+		default:
+			return nil, err
+		}
+	}
+
+	sessionId := uuid.New().String()
+
+	annotations := map[string]string{"flightctl.io/console": sessionId}
```

Comment:
Please declare a const in store/model/device.go
According to the format we've been using, it would be device-controller/console.